v2/emailpassword/quick-setup/frontend.mdx

```diff
@@ -138,10 +138,14 @@ SuperTokens.init({
 <AppInfoForm askForAPIDomain>
 
 ```tsx
-import supertokens from "supertokens-website-script";
+import supertokens from "supertokens-web-js-script";
 supertokens.init({
-    apiDomain: "^{form_apiDomain}",
-    apiBasePath: "^{form_apiBasePath}"
+    appInfo: {
+        apiDomain: "^{form_apiDomain}",
+        apiBasePath: "^{form_apiBasePath}",
+        appName: "...",
+    },
+    recipeList: [],
```

Comment:
need to do session.init here?

Reply:
Done in a different PR: https://github.com/supertokens/docs/pull/347/files#diff-a6fb346d682426623bbf3035d4196d1dae0d1189d6fffa8885d362e0dc817865